Documentation/NetworkFlowLogging/ama-metrics-prometheus-config-node.yaml

```diff
@@ -0,0 +1,32 @@
+kind: ConfigMap
+apiVersion: v1
+data:
+  prometheus-config: |-
+    scrape_configs:
+    - job_name: ama-logs-daemonset
+      kubernetes_sd_configs:
+        - role: pod
+      relabel_configs:
```

Comment:
This file can be removed. For retina network flow logs public doc, we can use this - https://aka.ms/ci-qos-config

Reply:
removed